generic/samples/native-client/src/nativeMain/kotlin/app.kt

```diff
@@ -0,0 +1,9 @@
+import io.ktor.client.engine.curl.Curl
```

Comment:
Not clear, do we suggest users to use star import or not? We have stars in runner.kt and we don't have it here

Reply:
I think we generally should recommend same as Kotlin and default to *